charts/woodpecker/README.md

```diff
@@ -16,6 +16,13 @@ helm install woodpecker woodpecker/woodpecker
 **Note**: The `woodpecker/woodpecker` chart contains both the server and the agent.
 If you want to install only the server or agent you can use the [`woodpecker/server`](./charts/server/README.md) or [`woodpecker/agent`](./charts/server/README.md) charts respectively.
 
+### Secrets
+
+Since version 1.1.0 the chart bootstrap a secret for `WOODPECKER_AGENT_TOKEN` which is required for the communication between agent and server.
```

Comment:
```suggestion
Since version 1.1.0 the chart bootstrap a secret for `WOODPECKER_AGENT_TOKEN` which is required for the communication between the agent and server.
```

Reply:
There could be multiple agents. So maybe the best wording could be

```suggestion
Since version 1.1.0 the chart bootstrap a secret for `WOODPECKER_AGENT_TOKEN` which is required for the communication between agent(s) and server.
```

I don't think it needs a "the" in here.